daphne_service_utils/dapf/src/main.rs

```diff
@@ -431,6 +452,50 @@ async fn main() -> Result<()> {
             println!("ID of the new HPKE config: {hpke_config_id}");
             Ok(())
         }
+        Action::Aggregate {} => {
+            let t = dapf::acceptance::Test::from_env()?;
```

Comment:
A mixture of environment variables and command line arguments is confusing from a UX perspective. Let's consider which of these we want to be environment variables and which we don't.